packages/app-project/src/helpers/getStaticPageProps/getStaticPageProps.spec.js

```diff
@@ -58,50 +58,13 @@ describe('Helpers > getStaticPageProps', function () {
     }
   }
 
-  const availableSubjects = {
-      1: 4,
-      2: 10,
-      3: 10
-  }
-
   const completeness = {
     1: 0.6,
     2: 0,
     3: 0
   }
```

Comment:
This isn't used any more in these tests, but the `UserPersonalization` tests fail if I delete it.